Source/NaturalLanguageClassifierV1/Models/ClassifierDetails.swift

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright IBM Corporation 2015
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Freddy
+
+extension NaturalLanguageClassifierV1 {
+    
+    public struct ClassifierDetails: JSONDecodable {
+        
+        /// Unique identifier for this classifier
+        public let classifierId: String
+        
+        /// User-supplied name for the classifier
+        public let name: String?
+        
+        /// The language used for the classifier
+        public let language: String
+        
+        /// Date and time (UTC) the classifier was created
+        public let created: String
```

Comment:
I like that you made the date a `String`. I'm still iffy about it, but definitely lean towards using `String` instead of `NSDate` to protect against future changes to the service.

Oh, on the flip-side, though, if our single repository ever gets split up into a repository for each service, then we probably _do_ want to parse these properties as `NSDate` since it's easier to update the version tag. No need to worry about that now, though.